tests/countries/test_palestine.py

```diff
@@ -0,0 +1,437 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from unittest import TestCase
+
+from holidays.constants import CATHOLIC, ORTHODOX
+from holidays.countries.palestine import Palestine, PS, PSE
+from tests.common import CommonCountryTests
+
+
+class TestPalestine(CommonCountryTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        years = range(1989, 2050)
+        super().setUpClass(Palestine, years=years)
+        cls.catholic_holidays = Palestine(categories=CATHOLIC, years=years)
+        cls.orthodox_holidays = Palestine(categories=ORTHODOX, years=years)
+        cls.no_estimated_holidays = Palestine(years=years, islamic_show_estimated=False)
+
+    def test_country_aliases(self):
+        self.assertAliases(Palestine, PS, PSE)
+
+    def test_no_holidays(self):
+        self.assertNoHolidays(Palestine(categories=self.holidays.supported_categories, years=1988))
+
+    def test_new_years_day(self):
+        name = "رأس السنة الميلادي"
+        self.assertHolidayName(name, (f"{year}-01-01" for year in range(1989, 2050)))
+        self.assertHolidayName(
+            name, self.catholic_holidays, (f"{year}-01-01" for year in range(1989, 2050))
+        )
+
+    def test_orthodox_new_years_day(self):
+        name = "عيد رأس السنة الشرقي"
+        self.assertHolidayName(
+            name, self.orthodox_holidays, (f"{year}-01-14" for year in range(1989, 2050))
+        )
+
+    def test_womens_day(self):
+        name = "يوم المراة العالمي"
+        self.assertHolidayName(name, (f"{year}-03-08" for year in range(1989, 2050)))
```

Comment:
```suggestion
        self.assertHolidayName(
            "يوم المراة العالمي", (f"{year}-03-08" for year in range(1989, 2050))
        )
```
Same for similar holidays.